src/Client/ClientBase.cpp

```diff
@@ -2692,8 +2725,17 @@ void ClientBase::init(int argc, char ** argv)
         queries_files = options["queries-file"].as<std::vector<std::string>>();
     if (options.count("multiline"))
         config().setBool("multiline", true);
+
     if (options.count("multiquery"))
+    {
+        auto queryText = options["multiquery"].as<std::string>();
         config().setBool("multiquery", true);
+        if (!queryText.empty() && !options.count("query"))
+        {
```

Comment:
(let's omit parentheses for single line ifs)

Reply:
done: this code no longer actual